lab1/Pasichnichenko_Task1.py

```diff
@@ -0,0 +1,12 @@
+# 3.1
+print("'Multi-paradigm programming languages':Task 1")
+print("Pasichnichenko Illia IKM-221a")
+
+# 3.2
+for i in range(45):
```

Comment:
please look through these links and try to refactor your code:
https://www.w3schools.com/python/python_lists_comprehension.asp
https://www.w3schools.com/python/ref_string_join.asp